src/main/java/org/prebid/server/health/model/StatusResponse.java

```diff
@@ -0,0 +1,15 @@
+package org.prebid.server.health.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Value;
+
+import java.time.ZonedDateTime;
+
+@AllArgsConstructor(staticName = "of")
+@Value
+public class StatusResponse {
```

Comment:
Consider using `Map<String, Object>` instead of this model.
It might be more universal for cases when status must be configurable string, or for ex. if we don't need to know last update date/time.
So, we can create `ApplicationHealthChecker` with just "ready" status.

Let's think about it.